common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -225,12 +228,12 @@ class @Problem
         flag = false
     return flag
 
-  # Review button click brings user back to top of problem
-  review_question_click: =>
+  # Scroll to problem metadata and next focus is problem input
+  scroll_to_problem_meta: =>
     $('html, body').animate({
       scrollTop: @questionTitle.offset().top
     }, 500);
-    @questionTitle.focus();
+    @questionTitle.focus()
```

Comment:
So you are calling focus here, but for some reason keyboard focus is still not moving.